tests/test_rich_markup_mode.py

```diff
@@ -0,0 +1,40 @@
+import typer
+import typer.completion
+from typer.testing import CliRunner
+
+runner = CliRunner()
+rounded = ["╭", "─", "┬", "╮", "│", "├", "┼", "┤", "╰", "┴", "╯"]
+
+
+def test_rich_markup_mode_none():
+    app = typer.Typer(rich_markup_mode=None)
+
+    @app.command()
+    def main(arg: str):
+        """Main function"""
+        print(f"Hello {arg}")
+
+    assert app.rich_markup_mode is None
+
+    result = runner.invoke(app, ["World"])
+    assert "Hello World" in result.stdout
+
+    result = runner.invoke(app, ["--help"])
+    assert all(c not in result.stdout for c in rounded)
+
+
+def test_rich_markup_mode_rich():
```

Comment:
This test fails with 0.12.5 release
```
def test_rich_markup_mode_rich():
        app = typer.Typer(rich_markup_mode="rich")

        @app.command()
        def main(arg: str):
            """Main function"""
            print(f"Hello {arg}")

        assert app.rich_markup_mode == "rich"

        result = runner.invoke(app, ["World"])
        assert "Hello World" in result.stdout

        result = runner.invoke(app, ["--help"])
>       assert any(c in result.stdout for c in rounded)
E       assert False
E        +  where False = any(<generator object test_rich_markup_mode_rich.<locals>.<genexpr> at 0x7f931ae59d80>)

tests/test_rich_markup_mode.py:40: AssertionError
```

Reply:
Hi @kraj, thanks for the comment. That's odd - I can't reproduce the test failure locally and the CI goes green with this change as well.

Could you please open a new [discussion thread](https://github.com/fastapi/typer/discussions) to look into this more, and provide us there with more information about your system setup and the failure you're seeing? 🙏

Reply:
I build a minimal image which just have a kernel+busybox+typer and its dependencies. So perhaps thats the difference, some implicit dependency coming in indirectly on full distributions like ubuntu, fedora etc in CI